js/pokemap.js

```diff
@@ -4,87 +4,53 @@ var util = require('util');
 var L = require('leaflet');
 var io = require('socket.io-client');
 var socket = io.connect("http://localhost:3000");
-var Pokemon = require('./basictypes');
 var config = require('./config');
 require('leaflet.locatecontrol');
+require('leaflet-routing-machine');
+require('leaflet-control-geocoder');
 
 // Include stylesheets
 require('../css/style.css');
 require('leaflet/dist/leaflet.css');
 
 var mymap = null;
-var apiURL={};
-var getAllSightingsURL = "/api/pokemon/sighting";
-var getAllSightingsByTimeRangeURL = "/api/pokemon/sighting/ts/";
-var getAllPokemon = "/api/pokemon";
-var getPokemonById = "/api/pokemon/id/";
+var apiEndpoint=null;
+var getAllPokemon = "pokemon/";
+var getAllSightings = getAllPokemon + "sighting/";
+var getAllSightingsByTime = getAllSightings + "ts/";
+var getPokemonById = getAllPokemon + "id/";
 var getAllPredictions = {};
 
-var PokeMap = function(htmlElement, options={filter : {pokemonIds: 0, sightingsSince: 0, predictionsUntil: 0}, tileLayer: config.currentMap,apiEndPoint : 'http://pokedata.c4e3f8c7.svc.dockerapp.io:65014'}) {
+var PokeMap = function(htmlElement, options={filter : {pokemonIds: 0, sightingsSince: 0, predictionsUntil: 0}, tileLayer: config.currentMap,apiEndpoint : 'http://pokedata.c4e3f8c7.svc.dockerapp.io:65014/api/'}) {
```

Comment:
This is incorrect. The `apiEndpoint` will default to `http://pokedata.c4e3f8c7.svc.dockerapp.io:65014` so you have to append the `/api` yourself where necessary.